app/presenters/user_analytics_presenter.rb

```diff
@@ -45,41 +44,72 @@ def display_percentage(numerator, denominator)
 
   private
 
+  attr_reader :current_facility
+
+  def diabetes_enabled?
+    FeatureToggle.enabled?('DIABETES_SUPPORT_IN_PROGRESS_TAB') && current_facility.diabetes_enabled?
+  end
+
   def daily_stats
+    diabetes_enabled? ? daily_total_stats : daily_htn_stats
+  end
+
+  def daily_htn_stats
+    follow_ups =
+      current_facility
+        .hypertension_follow_ups(:day, last: DAYS_AGO)
```

Comment:
Should we rename these methods to `hypertension_follow_ups_by_period`? Contrasted against the registrations calculation below, it might be more intuitive?